python/triton/language/semantic.py

```diff
@@ -226,7 +226,6 @@ def fdiv(input: tl.tensor,
         raise ValueError("both operands of fdiv must have floating poscalar type")
     input, other = binary_op_type_checking_impl(input, other, builder, False, False, False, True)
     ret = builder.create_fdiv(input.handle, other.handle)
-    ret.set_fdiv_ieee_rounding(ieee_rounding)
```

Comment:
If we delete this line, then how do we plan to handle `set_fdiv_ieee_rounding` in Triton-MLIR? This is a pretty important flag

Reply:
arith::DivFOp doesn't support ieee_rounding, how about switch back to inline ptx?

Reply:
Yeah, we probably want our own triton.fdivop then. I assume `DivFOp` has ieee rounding, which is slow on Nvidia hardware -- we want to use div.full or even div.approx there. Could you add a `triton::DivFOp` that takes an `ieee_rounding` attribute?

Reply:
Sure, let me do this.